python/src/deltachat/testplugin.py

```diff
@@ -13,6 +13,7 @@
 
 import pytest
 import requests
+import py
```

Comment:
https://pypi.org/project/py/ description says to use https://docs.python.org/3/library/pathlib.html instead

Reply:
yes but usage is minimal, and easy enough to get ported if needed.  While i co-authored py, i didn't co-author pathlib :) and as we are not using in the app or in the binbdings, it's fine to keep using py.

Reply:
(pytest has a dependency on py)

Reply:
k, i ported it.  it's more code and it took me a bit to learn the pathlib API. Requires a bit more calls than what i like for this 2-3 line code thing.  Also i think moving away from tmpdir/py.path.local is going to take quite a while anyway. Many of our tests use it and i wouldn't start the porting until it's really needed.